src/common.ts

```diff
@@ -1,6 +1,6 @@
-export class ValidationError extends Error {
-    constructor(public message: string, public path: string) {
-        super(message);
-        Object.setPrototypeOf(this, ValidationError.prototype);
-    }
+export class ValidationError {
```

Comment:
We don't want to throw exceptions and stop the process, we want to collect all validation errors giving the user the chance to address them all in one.

Reply:
love this!